### ISSUE TITLE
create own logfile for piLogger-cloud plugin

### ISSUE THREAD

**Issue by do3meli** [maglub/piLogger#107]
currently the piLogger-cloud plugin is not having its own logfile

**Comment by maglub** [comment]
It sure does have it's own logfile in my version in branch "develop", which
I have not had the time to merge yet:

```
pi@1wire-r001fry ~/piLogger/remote-logging.d $ ls -la /var/log/piLogger/ |
grep piLogger-cloud
-rw-r--r--  1 pi   pi    207676 Sep 23 15:35 piLogger-cloud.log
```

If you use the function "logIt" you will automatically write to it.

//magnus

**Comment by maglub** [comment]
Resolved by 72257fc653aedf1c841f6a09ec1c5edd8ead7da6
